packages/components/forms/src/index.ts

```diff
@@ -1,10 +1,3 @@
-export * from './Label';
-export * from './controlled-input-field';
-export * from './checkbox-field';
-export * from './radio-button-field';
-export * from './controlled-input';
+export * from './radio';
 export * from './checkbox';
-export * from './switch';
-export * from './radio-button';
-export * from './base-input';
-export * from './base-checkbox';
```

Comment:
👀 

All these components can be deleted?
Is that reason why parcel is failing in my branch then? hahahah

Reply:
yeah, some were the old migrated version, and some where created on the `baseCheckbox` PR, but after the discussion we had, we don't need all of them anymore.

Reply:
oh, the pure joy of solving conflicts 😄